### PR TITLE
[docs] Add manage packages docs

### PR DESCRIPTION
**Pull request by Jennifer88huang**
### Motivation
In [PIP-50](https://github.com/apache/pulsar/wiki/PIP-50%3A-Package-Management), we introduce a new way to manage packages.

### Modifications

Add related doc content based on the PIP and [REST API](https://pulsar.apache.org/packages-rest-api/?version=master&apiversion=v3#operation/listPackages).

### CODE REVIEW

**Comment by Jennifer88huang** [comment]
@zymap @tuteng could you please help check whether the commands / REST APIs are right? Thank you very much.

File: site2/docs/admin-api-packages.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+|`type` |The type of the package. The following types are supported: `function`, `sink` and `source`. |
+| `name`|The fully qualified name of the package: `<tenant>/<namespace>/<package name>`.|
+|`version`|The version of the package.|
+
+The following is a code sample.
+
+```java
+class PackageName {
+   private final PackageType type;
+   // REST API handlers.
```
Comment (zymap) [inline site2/docs/admin-api-packages.md:24]:
```suggestion

```

**Review by zymap — APPROVED** [review]

**Review by zymap — COMMENTED** [review]
Overall looks good to me. Thanks for working on this.

**Comment by Jennifer88huang** [comment]
@zymap Thank you very much for your support~

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**